server/notification-providers/home-assistant.js

```diff
@@ -11,7 +11,7 @@ class HomeAssistant extends NotificationProvider {
 
         try {
             await axios.post(
-                `${notification.homeAssistantUrl}/api/services/notify/${notificationService}`,
+                `${notification.homeAssistantUrl.replace(/\/*$/, "")}/api/services/notify/${notificationService}`,
```

Comment:
`.trim()` should be better?

Reply:
`.trim()` won't strip trailing slashes, just whitespace. Trimming before stripping would be a good idea though, I'll add.

Reply:
Understood, I wrongly read the title.